README.md

```diff
@@ -89,7 +89,7 @@ public class MainApplication extends Application implements ReactApplication {
 
 1. Run `npm install react-native-mail --save`
 2. Open your project in XCode, right click on `Libraries` and click `Add
-   Files to "Your Project Name"` [(Screenshot)](http://url.brentvatne.ca/jQp8) then [(Screenshot)](http://url.brentvatne.ca/1gqUD).
+   Files to "Your Project Name"` [(Screenshot)](http://url.brentvatne.ca/jQp8) then [(Screenshot)](https://github.com/pedramsaleh/react-native-mail/blob/master/add-xcodeproj.png?raw=true).
```

Comment:
Yeah I wanted to do that but it wouldn't let me upload the screenshot to the main repo to get the URL and reference it that way. I also tried referencing it by relative path so it would still work that when if/when merged, but then it doesn't just display the picture; it also puts a github menu around it. Not sure how to fix it without hosting externally.